From: Luigi Baldoni <aloisio@gmx.com>
Date: 2018-02-09 16:58:22 +0100
Subject: Add support for extra hardware

Add support for UniTrend UT61B/UT61D and Metex M-3840D
that had been added in 0.9.2 post-fork.

Index: QtDMM-0.9.8/src/sources/dmmprefs.cpp
===================================================================
--- QtDMM-0.9.8.orig/src/sources/dmmprefs.cpp
+++ QtDMM-0.9.8/src/sources/dmmprefs.cpp
@@ -81,6 +81,7 @@ struct DMMInfo dmm_info[] = {
 
 							  {"Metex M-3660D", 1, 0, 7, 2, 1, 0, 1, 0, 0,1,1,1},
 							  {"Metex M-3830D", 1, 0, 7, 2, 4, 0, 1, 0, 0,1,1,1},      // no image
+							  {"Metex M-3840D", 1, 0, 7, 2, 4, 0, 1, 0, 0,1,1,1},
 							  {"Metex M-3850D", 1, 0, 7, 2, 4, 0, 1, 0, 0,1,1,1},
 							  {"Metex M-3850M", 5, 0, 7, 2, 4, 0, 1, 0, 0,1,1,1},
 							  {"Metex M-3870D", 1, 0, 7, 1, 1, 0, 1, 0, 0,1,1,1},
@@ -112,6 +113,8 @@ struct DMMInfo dmm_info[] = {
 
 							  {"Uni-Trend UT30A", 3, 5, 8, 1, 1, 0, 1, 0, 0,1,1,1},
 							  {"Uni-Trend UT30E", 3, 5, 8, 1, 1, 0, 1, 0, 0,1,1,1},   // no image
+							  {"Uni-Trend UT61B", 3, 8, 8, 1, 1, 0, 1, 0, 0,1,1,1},   // no image
+							  {"Uni-Trend UT61D", 3, 8, 8, 1, 1, 0, 8, 0, 0,0,0,1},   // no image
 
 							  {"Voltcraft M-3610D", 1, 0, 7, 2, 1, 0, 1, 0, 0,1,1,1},  // no image
 							  {"Voltcraft M-3650D", 1, 0, 7, 2, 1, 0, 1, 0, 0,1,1,1},
